src/module.c

```diff
@@ -356,6 +356,7 @@ typedef struct RedisModuleServerInfoData {
 #define REDISMODULE_ARGV_SCRIPT_MODE (1<<6)
 #define REDISMODULE_ARGV_NO_WRITES (1<<7)
 #define REDISMODULE_ARGV_CALL_REPLIES_AS_ERRORS (1<<8)
+#define REDISMODULE_ARGV_NO_DENY_OOM (1<<9)
```

Comment:
Maybe call it `REDISMODULE_ARGV_RESPECT_DENY_OOM`, when I first read it, it was look like this flag indicate to ignore OOM...